src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -90,6 +88,7 @@ class JesBackendCall(val backend: JesBackend,
   override def poll(previous: ExecutionHandle)(implicit ec: ExecutionContext) = Future {
     previous match {
       case handle: JesPendingExecutionHandle =>
+        logger.debug(s"Polling JES Job ${handle.run.runId}.")
```

Comment:
does this / can this include the workflow ID and call name / index in the output?